test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -429,6 +429,37 @@ TEST_F(ClusterManagerImplTest, RingHashLoadBalancerV2Initialization) {
   create(parseBootstrapFromV2Yaml(yaml));
 }
 
+// Test that the cluster manager correctly re-creates the worker local LB when there is a host
+// set change.
+TEST_F(ClusterManagerImplTest, RingHashLoadBalancerThreadAwareUpdate) {
+  const std::string json = fmt::sprintf(
+      R"EOF(
+  {
+    %s
+  }
```

Comment:
Nit: seems a bit verbose, `fmt::sprintf("{%s}", clustersJson...`.

Reply:
I just copied this, will fix.